src/App.tsx

```diff
@@ -17,18 +80,33 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onChangeStatus={handleChangeStatus}
+                query={query}
+                onChangeQuery={handleChangeQuery}
+                onReset={onReset}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loading && <Loader />}
+              <TodoList
+                todos={filteredTodos}
+                showSelectedTodo={handelTodo}
```

Comment:
The function `handelTodo` is used here, but it contains a typo. It should be `handleTodo` to match the corrected function name.